lstchain/reco/disp.py

```diff
@@ -46,7 +47,13 @@ def disp(cog_x, cog_y, src_x, src_y):
         else:
             disp_angle = np.arctan(disp_dy/disp_dx)
 
-    disp_sign = np.sign(disp_dx)
+    # disp_sign : indicates in which direction, "positive" or "negative" we must move along the 
+    # reconstructed image axis (with direction defined by the versor cos(hillas_psi), sin(hillas_psi)) 
+    # we must move from cog_x, cog_y to get closest to the true direction (src_x, src_y)
+
+    sqrdist_plus  = (cog_x + disp_norm*np.cos(hillas_psi) - src_x)**2 + (cog_y + disp_norm*np.sin(hillas_psi) - src_y)**2
```

Comment:
This is a little hard to read...

Is it the same as here?

https://github.com/fact-project/aict-tools/blob/ee091fc933110ff008e32df02ee12a97e412e8b3/aict_tools/preprocessing.py#L42-L79

Reply:
Looks different....

Reply:
Same result, but ok, the approach you mention is simpler. Will change it.